src/Tables.jl

```diff
@@ -159,7 +159,7 @@ isrowtable(::Type{<:DictRows}) = true
 schema(x::DictRows) = nothing
 
 # Dict of AbstractVectors for Tables.columns
-const DictColumns = AbstractDict{K, V} where {K <: Union{Symbol, String}, V <: AbstractVector}
+const DictColumns = D where {D <: Union{AbstractDict{<:AbstractString, <:AbstractVector}, AbstractDict{Symbol, <:AbstractVector}}}
```

Comment:
Note that this changes additionally previous behavior (which I think was incorrect - now `false` will be returned):
```
julia> Tables.istable(Dict{Union{String, Symbol}, Vector})
true
```

Reply:
Isn't this just:
```suggestion
const DictColumns = Union{<:AbstractDict{<:AbstractString, <:AbstractVector}, <:AbstractDict{Symbol, <:AbstractVector}}}
```

Reply:
It is :)